### DISCUSSION TITLE
功能建议：作者考虑添加Netlify这个托管功能嘛

### DISCUSSION THREAD

**Discussion by Litway0517** [Qexo/Qexo#268] [Ideas]
有意向添加，添加Netlify托管功能嘛？

**Comment by am-abudu** [comment]
没有这个考虑，但如果你有能力的话可以自行提交PR

**Comment by Fgaoxing** [comment]
@Litway0517 vercel和netlify的无服务支持的语言和框架不同，基本上无法实现

**Comment by Fgaoxing** [comment]
@Litway0517 其次是配置文件和运行方式，netlify我不是特别了解，但是据我所知vercel和netlify的无服务器配置文件的方式不太相同

**Reply by Litway0517** [reply]
嗯嗯，我也是这两天才接触netlify

**Comment by Raisetsu41** [comment]
主要是 netlify 在国内访问更友好

**Comment by am-abudu** [comment]
netlify目前还不支持自定义运行时。无法支持。
